flow/project.py

```diff
@@ -326,20 +345,98 @@ def cmd(self):
 
     def set_status(self, value):
         "Store the operation's status."
-        self.job._project.document.setdefault('_status', dict())[self.id] = int(value)
+        self._jobs[0]._project.document.setdefault('_status', dict())[self.id] = int(value)
```

Comment:
Are the jobs sorted in some way? What if the order changes? Are we still able to retrieve this? I think I'd prefer to store this in the project-doc.

Reply:
Since we're not yet introducing the actual aggregation, I don't think there is a string reason to implement this for this PR. The reason behind my comment is that storing is only helpful for status-check and I have implemented this is #335.
This PR only changes the internal workflow logic.

Reply:
Ok, that's fine. Can you add an in-code comment so that it is easier to keep track of this?

Reply:
I'll do that. Thank you